### ISSUE TITLE
Cascade does not work with specific syntax

### ISSUE THREAD

**Issue by Cactucs** [contember/contember#113]
When I send request below to API, it show have same result as `deleteDocument("XXXX")`. But the following mutation fails to delete Taxon entity (`8d52248c-eafb-48e4-b28e-69432726bec3`). This request is generated by Contember Admin. 

**Current behaviour:**
Only Document entity is deleted and the relation is set to null.  

**Desired behaviour:** 
Document and Taxon should be deleted. 

```
mutation {
	subTree_2061139282___59698211_436f_4f5f_b1f4_fec2356f36fd: updateTaxon(data: {category: {update: {children: [{update: {by: {id: "8d52248c-eafb-48e4-b28e-69432726bec3"}, data: {document: {delete: true}}}, alias: "_8d52248c_eafb_48e4_b28e_69432726bec3"}]}}}, by: {id: "59698211-436f-4f5f-b1f4-fec2356f36fd"}) {
		ok
		errorMessage
		node {
			id
		}
		validation {
			valid
			errors {
				path {
					__typename
					... on _FieldPathFragment {
						field
					}
					... on _IndexPathFragment {
						index
						alias
					}
				}
				message {
					text
				}
			}
		}
		errors {
			type
			message
			path {
				__typename
				... on _FieldPathFragment {
					field
				}
				... on _IndexPathFragment {
					index
					alias
				}
			}
		}
	}
}
```

Schema: https://github.com/skaut/znalostnibaze-api/tree/master/projects/gov/api/model (private)

**Comment by matej21** [comment]
note to self: deleting [this line](https://github.com/contember/contember/blob/42e1e44cedec2646b8c8ffa12d315c922f500052/packages/engine-content-api/src/sql/update/SqlUpdateInputProcessor.ts#L229) should be sufficient (also in one-has-one-owner relation)

**Comment by Cactucs** [comment]
This is fixed in #115, but not released yet. Correct?
